### DISCUSSION TITLE
Scalper

### DISCUSSION THREAD

**Discussion by Sol-au-fein** [secretcowlevel/DoomtrooperPublic#72] [Card Balance]
Why would you want Scalper, when you can have Ceremonial Blade. Buff it to be more interesting weapon.

Scalper (MISHIMA)
EQUIPMENT BLADE
F +3, Opponent's F -3

**Comment by david-roxygen** [comment]
True your thought about scalper vs mishimese ceremonial blade. But considering that it is common, also in the current version could be enough.
